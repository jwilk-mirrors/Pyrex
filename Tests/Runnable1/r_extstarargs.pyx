cdef class Swallow:

	def __init__(self, name, airspeed, *args, **kwds):
		print "Name:", name
		print "Airspeed:", airspeed
		print "Extra args:", args
		print "Extra keywords:", kwds
