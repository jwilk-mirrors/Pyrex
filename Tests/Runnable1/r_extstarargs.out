Name: Brian
Airspeed: 42
Extra args: ()
Extra keywords: {}
Name: Brian
Airspeed: 42
Extra args: ('African',)
Extra keywords: {}
Name: Brian
Airspeed: 42
Extra args: ()
Extra keywords: {}
Name: Brian
Airspeed: 42
Extra args: ()
Extra keywords: {'coconuts': 3, 'species': 'African'}
Name: Brian
Airspeed: 42
Extra args: ('African',)
Extra keywords: {'coconuts': 3}
